apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -91,7 +91,7 @@ defmodule Aecore.Structures.DataTx do
           chainstate
         end
 
-      Map.put(new_chainstate, :accounts, new_accounts_state)
+      Map.put(new_chainstate, :accounts, new_accounts_state_tree)
```

Comment:
Since you've made function for adding a value in the Account module, why not do the same with the ChainState module? Maybe inside the Chain.Worker for adding/updating values

Reply:
Your suggestion is good and could rise as an issue. The current issue is for a structure found inside the ChainState.